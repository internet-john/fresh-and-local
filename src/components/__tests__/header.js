import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../header';

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Header siteTitle="Fresh and Local" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
