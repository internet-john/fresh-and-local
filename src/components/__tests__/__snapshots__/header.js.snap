// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  style={
    Object {
      "background": "transparent",
      "color": "#000",
      "fontSize": "42px",
      "letterSpacing": "-7.5px",
      "lineHeight": "initial",
    }
  }
>
  <h1
    style={
      Object {
        "margin": 0,
      }
    }
  >
    <a
      href="/"
      style={
        Object {
          "color": "#000",
          "textDecoration": "none",
        }
      }
    >
      Fresh and Local
    </a>
  </h1>
</header>
`;
